SU2_CFD/include/variables/CNEMOEulerVariable.hpp

```diff
@@ -71,7 +71,8 @@ class CNEMOEulerVariable : public CVariable {
   MatrixType dTvedU; /*!< \brief Partial derivative of vib.-el. temperature w.r.t. conserved variables. */
   MatrixType eves;   /*!< \brief energy of vib-el mode w.r.t. species. */
   MatrixType Cvves;  /*!< \brief Specific heat of vib-el mode w.r.t. species. */
-
+  VectorType Gamma;  /*!< \brief Ratio of specific heats. */
+  

```

Comment:
why vector ? isn't it just a variable ?

Reply:
Since we loop though iPoint in SetPrimVar, the Gamma variable would be overwritten each iteration.  So instead, it is saved as a vector so we store its value at each point.

However, if there is a better solution, Im all ears :)

Reply:
aaaaaaaaah. sorry, my bad! got it now